Test comments, fixing ManageVirtualMachineAsync sample, renaming *Os* as *OS* in VM/VMSS

martinsawicki opened the pull request:
...also: enabling authfile reader to skip comments and empty lines (and other invalid lines)

compat breaks from beta5:
- renaming all withers that contain `*Os*` as `*OS*` to use the correct casing for 2-letter acronyms, per .NET guidelines, within `IVirtualMachine` and `IVirtualMachineScaleSet` models

Commit:
test comments

msftclas commented:
@martinsawicki,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

Commit:
fixing ManageVirtualMachineAsync sample/test

Commit:
Merge remote-tracking branch 'upstream/Fluent' into Fluent

Commit:
removing VM.IUpdate.WithOSDiskCaching and .WithOSDiskSizeInGB due to Managed Disk API change

Commit:
never mind...

Commit:
renaming WithOsDiskName as WithOSDiskName, WithOsDiskEncryptionSettings as WithOSDiskEncryptionSettings

Commit:
rename VM's WithSpecializedOsDisk as WithSpecializedOSDisk, WithSpecializedOsUnmanagedDisk a WithSpecializedOSUnmanagedDisk

hovsepm commented on Samples/ResourceManagement/Compute/ManageVirtualMachineAsync/Program.cs:
```diff
@@ -115,38 +115,19 @@ await linuxVM.Update()
                         .ApplyAsync();
 
-                Utilities.Log("Tagged Linux VM: " + windowsVM.Id);
+                Utilities.Log("Tagged Windows VM: " + windowsVM.Id);
```
This should be Linux - I'm tagging a Linux VM in the sample

martinsawicki replied:
ah, ok, the variable name just reads in a very misleading way...

martinsawicki replied:
..but then the variable is wrong. fixing that.

Commit:
renaming VM WithOsDiskStorageAccountType as WithOSDiskStorageAccountType(), WithOsDiskVhdLocation() as WithOSDiskVhdLocation()

Commit:
cosmetic fix in sample

Commit:
renames in IVirtualMachineScaleSet:
- WithOSDiskName
- WithOSDiskName
- WithOSDiskCaching
- WithOSDiskStorageAccountType

Commit:
enabling authfile reader to skip blank lines and comments
